lib/matplotlib/widgets.py

```diff
@@ -229,7 +229,85 @@ def disconnect(self, cid):
         self._observers.disconnect(cid)
 
 
-class Slider(AxesWidget):
+class SliderBase(AxesWidget):
+    """
+    The base class for constructing Slider widgets. Not intended for direct
+    usage.
+
+    For the slider to remain responsive you must maintain a reference to it.
+    Call :meth:`on_changed` to connect to the slider event.
```

Comment:
This line is the cause of the doc failutres because `SliderBase` doesn't have an `on_changed` method. Is it preferable to just remove the `meth` directive or to remove the whole sentence?

Reply:
I removed it - The fact that `SliderBase` doesn't have an `on_changed` method probably should mean it's not responsible for documenting that it has that method...